### PR TITLE
Add multiple PHP versions to Github Actions

### PR DESCRIPTION
**Pull request by arturasfio**

### CODE REVIEW

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -9,13 +9,16 @@ on:
 jobs:
   phpunit-tests:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: ['7.2', '7.3', '7.4']
```
Comment (garas) [inline .github/workflows/php.yml]: Is SDK compatible with 8.0 and 8.1?

Comment (shababcardinity) [inline .github/workflows/php.yml]: wont 7.2 fail, our min is 7.2.5

Comment (arturasfio) [reply .github/workflows/php.yml]: SDK is compatible with 8.0. On 8.1 some deprecation notices are thrown, as mentioned in #26

Comment (arturasfio) [reply .github/workflows/php.yml]: 8.0 was added to the PHP version list. 8.1 will be added in scope of other PR, together with the deprecation notice fixes.

**Review by shababcardinity — APPROVED** [review]

**Pull request merged**